build.Dockerfile

```diff
@@ -21,10 +21,10 @@ WORKDIR /go/src/github.com/containous/traefik
 
 COPY glide.yaml glide.yaml
 COPY glide.lock glide.lock
-RUN glide install -v
+RUN glide install --strip-vendor
```

Comment:
-v is the shortcut for --strip-vendor, it makes sense to use the full flag for documentation...